src/components/TodoList.tsx

```diff
@@ -24,9 +26,19 @@ const TodoList: React.FC<TodoListProps> = ({ todos }) => {
         }
     }
 
```

Comment:
To handle saving data to localeStorage we could use useffect like this:
  useEffect(() => {
    setTodos(todoList);
  }, [todoList]);
  This way data would be saved to localStorage every time todoList state is changes.